ci/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+pipeline {
+    agent any
+    stages {
+        stage('Build') {
+            steps {
+                sh 'mvn clean install'
+            }
+        }
+        stage('Test') {
+            steps {
+                sh 'mvn test'
+            }
+         }
+        stage('Copy') {
+            steps {
+                 sh '''#!/bin/bash
+                 mkdir -p chatbot-backend
+                 cp -r ./api/target/api-0.0.1.jar ./chatbot-backend/
+                 cp -r ./integration/target/integration-0.0.1.jar ./chatbot-backend/
+                 '''
+            }
+        }
+
+        stage('Deploy') {
+            steps {
+                sh '''#!/bin/bash
```

Comment:
--httpPort mamy ustawiony application.yml, tutaj jest więc niepotrzebny, wywalmy to więc.